congestion-control-charter.txt

```diff
@@ -85,10 +78,4 @@ congestion control in the Internet, including assessments of compliance with
 existing standards. Other venues, e.g., IRTF, may be more appropriate for
 publishing such documents.
 
-Real-time, media adaptation algorithms for peer to peer communications remain
-the focus of RMCAT, but are in scope after that WG closes.
-
-Once the chartered deliverables are complete, CONGRESS will not remain open
-simply “in case” further work comes along. However, if the working group has
-adopted further work in accordance with the guidelines above, it can recharter,
-add milestones for them, and continue until that work is complete.
+CONGRESS will not remain open simply “in case” further work comes along. 

```

Comment:
It addresses my DISCUSS albeit the sentence looks a little weird like it is.

Reply:
Thanks.